src/types/Todo.ts

```diff
@@ -0,0 +1,12 @@
+export interface Todo {
```

Comment:
No major issues found. The code does not have any `// TODO:` comments and uses soft tabs set to 2 spaces. The interface definitions are clear and easy to understand. Good job! 👍